sensing/pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp

```diff
@@ -83,7 +124,7 @@ void DistortionCorrectorComponent::onPointCloud(PointCloud2::UniquePtr points_ms
   tf2::Transform tf2_base_link_to_sensor{};
   getTransform(points_msg->header.frame_id, base_link_frame_, &tf2_base_link_to_sensor);
 
-  undistortPointCloud(velocity_report_queue_, tf2_base_link_to_sensor, *points_msg);
+  undistortPointCloud(tf2_base_link_to_sensor, *points_msg);
 
   if (points_sub_count > 0) {
```

Comment:
At line 120, we confirm that `points_sub_count` is less than 1.
```
  if (points_sub_count < 1) {
    return;
  }
```
So, I think that this if-statement at 129 line is unnecessary.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/1120/commits/510fb12f2b5df5cc8dd1c5588990887d177e8bf8
Reflected!